zerver/management/commands/delete_realm.py

```diff
@@ -0,0 +1,45 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+from typing import Any
+
+from django.conf import settings
+from django.core.management.base import CommandError
+
+from zerver.lib.management import ZulipBaseCommand
+from zerver.models import Message, UserProfile
+
+
+class Command(ZulipBaseCommand):
+    help = """Script to deactivate a realm."""
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        self.add_realm_args(parser, True)
+
+    def handle(self, *args: Any, **options: str) -> None:
+        realm = self.get_realm(options)
+        assert realm is not None  # Should be ensured by parser
+
+        user_count = UserProfile.objects.filter(
+            realm_id=realm.id,
+            is_active=True,
+            is_bot=False,
+        ).exclude(
+            role=UserProfile.ROLE_GUEST,
```

Comment:
I would probably not exclude guests here -- they're real users for the purpose of this sort of sanity check. ("guest" users are just a permissions-limited user who can't access public streams, not something lighter that doesn't have real state).